ast/value.go

```diff
@@ -0,0 +1,85 @@
+package ast
+
+import (
+	"net/netip"
+
+	"github.com/cedar-policy/cedar-go/internal/ast"
+	"github.com/cedar-policy/cedar-go/types"
+)
+
+// Boolean creates a value node containing a Boolean.
+func Boolean[T bool | types.Boolean](b T) Node {
+	return wrapNode(ast.Boolean(types.Boolean(b)))
+}
+
+// True creates a value node containing True.
+func True() Node {
```

Comment:
Is there a motivation for not having singleton nodes: `True` and `False` ?

Reply:
We don't have special nodes for any of the other basic types (string, long), they all get put into a ValueNode.  So having a special node for True/False didn't occur to us.

Reply:
Oh and if you are meaning like `var True = ValueNode{Value:types.Boolean(true)}` ... that could work, but I'm not a huge fan of values that are supposed to be constant but could be changed by anyone.